### PR TITLE
Bank Customers Prediction

### PR DESCRIPTION
**Pull request by aryantiwari10**
## Related Issue 

- Info about the related issue 

- [x] LGM-SOC'21 Participant 
- [ ] DevIncept Participant
- [x] Contributor

Closes: #419

### Describe the add-ons or changes you've made

PROJECT TITLE : Bank Customers Prediction

GOAL : To predict the customers who are withdrawing their account from the bank due to some loss and other issues.

DATASET : https://www.kaggle.com/santoshd3/bank-customers

WHAT I HAD DONE : In this project first I performed a exploratory data analysis on the Bank Customer dataset which includes of data cleaning , data manipulation, data preprocessing , data visualization and after that I did training and testing , then I did the feature scaling and used ANN for the prediction. While in the EDA part I have included different plots for the different visualizations of our dataset .

LIBRARIES:

PANDAS
NUMPY
MATPLOTLIB
SEABORN
SCIPY
SKLEARN
Keras
TensorFlow

Conclusion : We get to know why customers are withdrawing their account from bank.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how it has been tested
Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (Only for Front End and UI/UX Designers)

 Original           | Updated
 :--------------------: |:--------------------:
 Original Screenshot | Updated Screenshot |

### CODE REVIEW

**Comment by Manasi2001** [comment]
What an impressive job on the notebook @aryantiwari10! The analysis is very well performed. Great work. 🎉

**Comment by aryantiwari10** [comment]
Thank you so much 😃😃
